### ISSUE TITLE
No warning when element HTML file is missing import

### ISSUE THREAD

**Issue by sigmundch** [dart-lang/polymer-dart#560]
_From @theqwertman on August 21, 2015 16:9_

For example, 

    <polymer-element name="address-entry">
      <template>
        <style>
          :host {
            display: block;
          }
        </style>
    <paper-dialog><h2>Something</h2></paper-dialog>

should report an error since there is no import link whatsoever, but the build succeeds without complaint, and the developer must investigate to find out why their element is not appearing.

_Copied from original issue: dart-lang/sdk#24167_

**Comment by sigmundch** [comment]
Is the problem in your example that you don't have the import to `paper-dialog` or the script tag that defines `address-entry`?

We used to have a linter that would give you warnings if you are missing the script tag or an import to find the definition of an element. However, since we added support for [`@HtmlImport` in Dart files](https://github.com/dart-lang/polymer-dart/wiki/Dart-to-HTML-imports), I am not sure if we can continue producing these warnings accurately (a missing import in the HTML is not always a problem).
